po/de.po

```diff
@@ -2276,12 +2276,18 @@ msgid ""
 "Entry column widths in Invoices, Bills, and Employee Vouchers may now be "
 "saved as defaults for each type of document."
 msgstr ""
+"Die Breite der Eingabespalte in Lieferatenrechnungen, Rechnungen und "
+"Mitarbeiterdokumente können jetzt als Standardwerte für jeden Dokumenttyp "
+"gespeichert werden."
 
 #: new_features-4.0.phtml:75
 msgid ""
 "Two menu items are added to the View menu when a Bill, Invoice, or Voucher "
 "tab has focus:"
 msgstr ""
+"Dem Menü \"Ansicht\" werden zwei Menüpunkte hinzugefügt, wenn die "
+"Registerkarte \"Lieferatenrechnung\", \"Rechnung\" oder \"Beleg\" den Fokus "
+"hat:"
```

Comment:
Siehe HTML in https://wiki.gnucash.org/wiki/GUI_Guidelines#Quoting_in_Strings. `<code>Ansicht</code>` wäre hier wohl zutreffend. Das sollten wir bereits in der Quelle einfügen.

Reply:
@fellen: ich finde die Funktion "Label" nicht, kannst du das entsprechend kennzeichnen?

Reply:
z.B. https://hosted.weblate.org/translate/gnucash/glossary/de/?q=state:empty
dann rechts in Quellinformtion auf den Stift neben Labels drücken.

Reply:
@fellen: ich habe da keinen Stift, daran sind bestimmt wieder die Berechtigungen schuld.